tmva/sofie/inc/TMVA/RModel.hxx

```diff
@@ -24,14 +24,20 @@ enum class Options {
    kDefault = 0x0,
    kNoSession = 0x1,
    kNoWeightFile = 0x2,
+   kTextWeightFile = 0x4,
```

Comment:
I don't think we need to add kTextWeightFile. This should be included in the kDefault option

Reply:
Done, yes we don't need `kTextWeightFile` since its already included in `kDefault`.